### DISCUSSION TITLE
Nested rows/column

### DISCUSSION THREAD

**Discussion by LukaszFormela** [orchidsoftware/platform#2060] [Q&A] [answered]
Hello, I'm struggling to make the design that looks as follows. I need to put one row ("Primary Information") directly above another ("Materials"), otherwise there's a lot of unused blank space as "Materials" is pushed to new line, just below "Additional Information".
Would it be possible to achieve it?

![scr](https://user-images.githubusercontent.com/39266504/149229735-5139cd25-5bcd-4919-bf24-fac11210c2a2.jpg)

**Comment by tabuna** [comment, marked as answer]
You can achieve the result like in the picture like this:

```php
public function layout(): array
{
    return [
        Layout::columns([
            [
                PrimaryInformationRow::class,
                MaterialsRow::class,
            ],
            AdditationalInformation::class,
        ]);
}
```

**Reply by LukaszFormela** [reply]
Wow, wish I knew that earlier :) What a great piece of software, hats off!
